vspace/src/main/java/edu/asu/diging/vspace/web/publicview/ExhibitionSpaceController.java

```diff
@@ -70,6 +70,7 @@ public String space(@PathVariable("id") String id, Model model) {
             model.addAttribute("alertType", "danger");
             model.addAttribute("messageType","invalidSpace");
             model.addAttribute("message", "Access Denied.");
+            model.addAttribute("isExhibition", true);
```

Comment:
Instead of this (why is it called `isExhibition` by the way?), I would forward to a 404 page.

Reply:
ok, the page should contain just image or header and footer as well?

Reply:
the user needs to be able to get back to the actual exhibition, so the header should be there. and the footer should be on every page.

Reply:
calling this isExhibition because this property will be only set for exhibition screen.

Reply:
this should be unnecessary once you forward to a dedicated page